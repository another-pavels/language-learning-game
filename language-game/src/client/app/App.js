import React from 'react';
import {render} from 'react-dom';

import GameContainer from './Game/GameContainer';
import {Grid, Header, Container} from 'semantic-ui-react'

export default class App extends React.Component {
    render () {
        return (
            <div>
                <GameContainer />
            </div>
        );
    }
}
