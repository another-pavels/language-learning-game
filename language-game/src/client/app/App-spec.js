import React from 'react';
import renderer from 'react-test-renderer';

import { createStore } from 'redux';
import { Provider } from 'react-redux'

import gameReducer from './Game/GameReducers';
import App from './App';

const rootReducer = gameReducer;

const store = createStore(
    rootReducer
);


describe('App (Snapshot)', () => {
  it('App renders', () => {
      const component = renderer.create(
          <Provider store={store}><App /></Provider>
      );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
