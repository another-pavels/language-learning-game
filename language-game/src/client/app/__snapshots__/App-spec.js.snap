exports[`App (Snapshot) App renders 1`] = `
<div>
  <div
    className="ui pushable segment">
    <div
      className="ui inverted ui sidebar top push visible menu">
      <div
        className="item"
        onClick={[Function]}>
        <h1
          className="ui inverted header">
          Mandarin Numbers
        </h1>
      </div>
    </div>
    <div
      className="pusher">
      <div
        className="ui basic segment">
        <div
          className="ui centered padded one column grid">
          <div
            className="column">
            <div
              className="ui text container">
              <div
                className="ui segment">
                <p>
                  I\'ll read off a series of random numbers. Type for me the numbers that you hear.
                </p>
                <button
                  className="ui massive button"
                  onClick={[Function]}
                  tabIndex={undefined}>
                  Begin
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
